internal/exec/exec.go

```diff
@@ -59,7 +65,14 @@ func executeCommand(c *command) error {
 
 		// nolint: gosec
 	cmd := exec.CommandContext(c.Ctx, c.Args[0], c.Args[1:]...)
-	cmd.Env = c.Env
+	cmd.Env = []string{}
+	for _, key := range passthroughEnvVars {
+		if value := os.Getenv(key); value != "" {
+			cmd.Env = append(cmd.Env, key+"="+value)
+		}
+	}
+	cmd.Env = append(cmd.Env, c.Env...)
```

Comment:
How would this work with duplicates - e.g. when `PATH` is declared explicitly in config and also passed through? I'm assuming that last one wins? Is it worth deduplicating it?

Reply:
afaik last one wins, so there is little to no use in deduplicating it 🤔